charts/k8ssandra/values.yaml

```diff
@@ -129,6 +129,35 @@ cassandra:
       #size:
       #newGenSize:
 
+  # Cassandra native transport ingress support
+  ingress:
+    # -- Enables Cassandra Traefik ingress definitions. Note that
+    # this is mutually exclusive with stargate.ingress.cassandra.enabled
+    enabled: false
+
+    # -- Determines which TCP-based ingress custom resources to template out. Currently only `traefik` is supported
+    method: traefik
+
+    # -- Optional hostname used to match requests. Warning: many native Cassandra clients, notably including cqlsh, initialize their connection by querying for
+    # the cluster's contactPoints, and thereafter communicate to the cluster using those names/IPs rather than whatever host was specified to the client. In
+    # order for clients to work correctly through ingress with a host filter, this means that the host filter must match the hostnames specified in the
+    # contactPoints. This value must be a DNS-resolvable hostname and not an IP address. To avoid this issue, leave this setting blank.
+    host:
+
+    traefik:
+      # -- Traefik entrypoint where traffic is sourced.
```

Comment:
This comment is referencing a property from the traefik chart installation. We should reference the docs to make it clear where it is coming from.

Reply:
Done.